assets/csidriveroperators/gcp-pd/01_namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-cluster-csi-drivers
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    openshift.io/node-selector: ""
```

Comment:
The `verify` job is failing because the bindata isn't updated, so we need a `make update` to fix that.